core/src/main/java/org/lflang/ast/IsEqual.java

```diff
@@ -99,6 +99,7 @@ public Boolean caseModel(Model object) {
   public Boolean caseImport(Import object) {
     return new ComparisonMachine<>(object, Import.class)
         .equalAsObjects(Import::getImportURI)
+        .equalAsObjects(Import::getImportPackage)
```

Comment:
Does this effectively function like "equal to importURI OR equal to importPackage"?

Reply:
@lhstrh Do you know how I can debug this function to fully understand its behavior?

Reply:
The comparison machine first checks whether the "other object" is an instance of the given class (see constructor). In order to give a definitive answer about equality, the other object and the given object are also compared in terms of what their getters return. If any of those comparisons yields `false`, the two objects must not be equal. You are correct to add this line. If you hadn't `caseImport` would have returned false positives (because it didn't look at the `getImportURI` getter).